frontend/src/pages/modelRegistry/screens/ModelRegistry.tsx

```diff
@@ -2,19 +2,25 @@ import React from 'react';
 import ApplicationsPage from '~/pages/ApplicationsPage';
 import { ProjectObjectType } from '~/concepts/design/utils';
 import TitleWithIcon from '~/concepts/design/TitleWithIcon';
-import ModelRegistryEmpty from '~/pages/modelRegistry/ModelRegistryEmpty';
 import { ModelRegistryContext } from '~/concepts/modelRegistry/context/ModelRegistryContext';
 import useRegisteredModels from '~/concepts/modelRegistry/apiHooks/useRegisteredModels';
 import RegisteredModelListView from './RegisteredModelListView';
+import EmptyRegisteredModels from './EmptyRegisteredModels';
+import RegisteredModelsTableToolbar from './RegisteredModelsTableToolbar';
 
 const ModelRegistry: React.FC = () => {
-  const { modelRegistries } = React.useContext(ModelRegistryContext);
+  const { preferredModelRegistry } = React.useContext(ModelRegistryContext);
   const [registeredModels, loaded, loadError] = useRegisteredModels();
 
   return (
     <ApplicationsPage
-      empty={modelRegistries.length === 0}
-      emptyStatePage={<ModelRegistryEmpty />}
+      empty={registeredModels.size === 0}
+      emptyStatePage={
+        <>
+          <RegisteredModelsTableToolbar />
```

Comment:
I'm gonna leave it for posterity, we should refactor this once we have the new redesign, I don't like this approach, we shouldn't render twice the component in the same view, In the near future we'll get the model registry in the header and the toolbar ONLY in the table